tests/test_series.py

```diff
@@ -1665,8 +1669,17 @@ def test_series_overloads_cat():
     check(assert_type(s.str.cat(sep=";"), str), str)
     check(assert_type(s.str.cat(None, sep=";"), str), str)
     check(
-        assert_type(s.str.cat(["A", "B", "C", "D", "E", "F", "G"], sep=";"), pd.Series),
-        pd.Series,
+        assert_type(
+            s.str.cat(["A", "B", "C", "D", "E", "F", "G"], sep=";"), UnknownSeries
+        ),
+        UnknownSeries,
+    )
+    check(
+        assert_type(
+            s.str.cat(pd.Series(["A", "B", "C", "D", "E", "F", "G"]), sep=";"),
+            UnknownSeries,
+        ),
+        UnknownSeries,
     )
```

Comment:
the return type of `cat` is the TypeVar `T`, so I think the intention might have been to preserve the input Series type?

anyway, i think that's out of scope for this PR, but I've opened https://github.com/pandas-dev/pandas-stubs/issues/1149to track it